config/settings/develop.py

```diff
@@ -7,6 +7,14 @@
 
 ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", "*").split(",")
 
+INSTALLED_APPS = INSTALLED_APPS + [  # noqa: F405
+    "debug_toolbar",
+]
+
+MIDDLEWARE = [
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
+] + MIDDLEWARE  # noqa: F405
+
```

Comment:
As DEBUG setting is env variable driven. Could this be moved under DEBUG setting/end of page with a condition setting?

```
if DEBUG :
    from .base import INSTALLED_APPS, MIDDLEWARE

    INSTALLED_APPS += [
        "debug_toolbar",
    ]

    MIDDLEWARE += [
        "debug_toolbar.middleware.DebugToolbarMiddleware",
    ]
```

Reply:
Additionally add in the condition - this was used to show the toolbar.

```
def show_toolbar(request) -> bool:
        return True

    DEBUG_TOOLBAR_CONFIG = {
        "SHOW_TOOLBAR_CALLBACK": show_toolbar,
        "SHOW_COLLAPSED": True,
    }
```

Reply:
That's all fine - please edit this PR

The only thing I'd say is that `"debug_toolbar.middleware.DebugToolbarMiddleware"` needs to be first on the list of `MIDDLEWARE`